### ISSUE TITLE
Abnormal crashes lead to shutdown on Folia 1.21-2b8c879

### ISSUE THREAD

**Issue by MineSunshineone** [PaperMC/Folia#262]
### Stack trace

```
[paste your stack trace or a paste.gg link here!](https://pastes.dev/P5FlpIJyMF)
```

### Plugin and Datapack List

plugins
[14:35:20 INFO]: Server Plugins (36):
[14:35:20 INFO]: Paper Plugins:
[14:35:20 INFO]: - MiraiMC
[14:35:20 INFO]: Bukkit Plugins:
[14:35:20 INFO]: - ajLeaderboards, AntiPopup, ArmorStandEditor, AxInventoryRestore, BetterGUI, ChestProtect, Chunky, CommandWhitelist, CoreProtect, CreeperConfetti
[14:35:20 INFO]: DeathMessage, Essentials, GSit, ICTTools, InvSeePlusPlus, Kaiivoid, KissKiss, LuckPerms, LushRewards, Matrix
[14:35:20 INFO]: MinePay, NoFlightInEnd, PlaceholderAPI, ProtocolLib, SkullPlugin, spark, TAB-Bridge, ToolStats, TrChat, UseTranslatedNames
[14:35:20 INFO]: voicechat, Whitelist4QQ, WorldEdit, ZAutoBroadcast, ZeroshiftFcmd

### Actions to reproduce (if known)

All of a sudden, the server shut down automatically, and I don't know

### Folia version

version
[14:34:06 INFO]: This server is running Folia version 1.21-DEV-dev/1.21@2b8c879 (2024-08-05T03:04:53Z) (Implementing API version 1.21-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21-DEV-2541ddf (MC: 1.21)

**Comment by MineSunshineone** [comment]
[2024-08-05-9.log](https://github.com/user-attachments/files/16491801/2024-08-05-9.log)
mybe helpful

**Comment by Spottedleaf** [comment]
Resolved in 1.21.3 https://github.com/PaperMC/Folia/commit/e1601057b3e8131127788aafc8f4e80b800556c3
